app/views/unattended/provisioning_templates/snippet/remote_execution_ssh_keys.erb

```diff
@@ -59,8 +59,8 @@ EOF
 
 <% if ssh_user != 'root' && host_param('remote_execution_effective_user_method') == 'sudo' -%>
 <% if @host.operatingsystem.family == 'Redhat' || @host.operatingsystem.family == 'Debian' -%>
-echo "<%= ssh_user %> ALL = (root) NOPASSWD : ALL
-Defaults:<%= ssh_user %> !requiretty" > /etc/sudoers.d/<%= ssh_user %>
+echo "<%= ssh_user %> ALL = "'(root)'" NOPASSWD : ALL" > /etc/sudoers.d/<%= ssh_user %>
```

Comment:
Why to have "'(root)'" instead of just
echo "<%= ssh_user %> ALL = (root) NOPASSWD : ALL" > /etc/sudoers.d/<%= ssh_user %>

Reply:
You are right, the quotes are superfluous.

Reply:
Rebased to develop and applied this suggestion. Ready for re-view @OttaviaB